ArmPkg/Drivers/ArmGic/ArmGicLib.c

```diff
@@ -225,10 +225,10 @@ ArmGicEndOfInterrupt (
 VOID
 EFIAPI
 ArmGicSetInterruptPriority (
-  IN UINTN  GicDistributorBase,
-  IN UINTN  GicRedistributorBase,
-  IN UINTN  Source,
-  IN UINTN  Priority
+  IN UINTN   GicDistributorBase,
+  IN UINTN   GicRedistributorBase,
+  IN UINTN   Source,
+  IN UINT32  Priority

```

Comment:
I think you also need to update the header file, see 
```
diff --git a/ArmPkg/Include/Library/ArmGicLib.h b/ArmPkg/Include/Library/ArmGicLib.h
index 93ce8aeb1994..ed0c18bbed71 100644
--- a/ArmPkg/Include/Library/ArmGicLib.h
+++ b/ArmPkg/Include/Library/ArmGicLib.h
@@ -213,7 +213,7 @@ ArmGicSetInterruptPriority (
   IN UINTN  GicDistributorBase,
   IN UINTN  GicRedistributorBase,
   IN UINTN  Source,
-  IN UINTN  Priority
+  IN UINT32  Priority
   );

 VOID

```

Reply:
Good catch, thank you!